tests/Drupal/Commenting/DocCommentUnitTest.inc.fixed

```diff
@@ -130,3 +130,18 @@ function test13() {
 function test14(array $matches, array $sub_key, $to) {
 
 }
+
+/**
+ * {@inheritdoc}
+ */
+function test15_this_inheritdoc_is_correct();
+
+/**
+ * {@inheritdoc}
+ */
+function test16_lower_case_fail_needs_braces();
+
+/**
+ * {@inheritDoc}
```

Comment:
The official docs say to use `{@inheritdoc}` with a lower case `d` https://www.drupal.org/docs/develop/standards/php/api-documentation-and-comment-standards#inheritdoc . I'm a bit on the fence if the fixer should produce wrong output. Ideally it would lower-case the `d` in the same go?

Don't think this is a blocker, if you agree please fix.

Reply:
I think I recall some discussion about which to use. You are right that the drupal standards page shows lowercase in the example, but it does not actually state that it must be lower case. 

I also found https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/5141 which talks about the camel case version being preferrable for PHPstan, and in https://docs.phpdoc.org/guide/guides/inheritance.html#inheritance all the examples are camel case. 

I'd be happy to change the fixer if Coder itself had a sniff to highlight that inheritDoc is wrong and should be changed to inheritdoc. But until that rule is decided, and a sniff created, I think it is best to leave the fixer to use what ever is currently in the file.

Reply:
I have created a small follow-up PR #185 to add a camel case example to the test file. This is in prep for a future sniff which may enforce one or the other. Having the four examples close in the file will make it simpler to adjust the fixed file and expected lines, rather than add the fourth case later, after other test examples may have been added at the end.